data/src/main/kotlin/com/ortin/gesturetranslator/data/db/UserRoomDB.kt

```diff
@@ -0,0 +1,25 @@
+package com.ortin.gesturetranslator.data.db
+
+import android.content.Context
+import androidx.room.Database
+import androidx.room.Room
+import androidx.room.RoomDatabase
+import com.ortin.gesturetranslator.data.entities.UserEntity
+
+@Database(
+    entities = [UserEntity::class],
+    version = 1
+)
+abstract class UserRoomDB: RoomDatabase() {
+    abstract fun userDao(): UserDao
+
+    companion object {
+        fun createDataBase(context: Context): UserRoomDB {
+            return Room.databaseBuilder(
+                context,
+                UserRoomDB::class.java,
+                "UserInfo.db"
```

Comment:
```suggestion
                "user_info"
```

Это стиль-код наименования баз данных в Android. `.db` добавлять не нужно, это уже заложено внутри. Иначе получится .db.db в конце файла)

Reply:
done